src/tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "outDir": "../lib"
     },
     "atom": {
-        "rewriteTsconfig": false
+        "rewriteTsconfig": true
```

Comment:
hm ok you're right, I'll revert this part. I forgot that our build expands filesGlob. This change would've alleviated a small inconvenience for Atom users (you can switch branches and not have to run `grunt` to keep coding), but it's something I'll have to live with